qa/rpc-tests/test_framework/mn_utils.py

```diff
@@ -0,0 +1,72 @@
+from .util import *
+
+class Masternode(object):
+    pass
+
+def prepare_mn(node, idx, alias):
```

Comment:
Now it is easy to merge because Dash has it the same way. But you that's right, this should be written as a class. Let's leave it for further improvement